libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -145,106 +155,155 @@ export const NavPaperWrapper = styled('div')`
   left: 0;
   right: 0;
   transform: translateY(100%);
+
+  ${({theme}) => css`
+    ${theme.breakpoints.up('md')} {
+      gap: ${theme.spacing(6)};
+      grid-row-gap: ${theme.spacing(12)};
+      grid-template-columns: 1fr 1fr;
+      padding: ${theme.spacing(2.5)} calc(100% / 6) calc(100% / 12);
+    }
+  `}
 `
 
-export const NavPaperCategoryList = styled('div')`
+export const NavPaperCategory = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(3)};
+  gap: ${({theme}) => theme.spacing(1)};
+  grid-auto-rows: max-content;
+`
+
+export const Name = styled('span')`
+  text-transform: uppercase;
+  font-weight: 300;
+  font-size: 14px;
+`
+export const Separator = styled('div')`
+  width: 100%;
+  height: 1px;
+  background-color: white;
 
   ${({theme}) => css`
     ${theme.breakpoints.up('sm')} {
-      grid-auto-flow: column;
-      grid-auto-columns: max-content;
+      display: none;
     }
   `}
 `
 
-export const NavPaperCategory = styled('div')`
+export const LinksGroup = styled('div')`
```

Comment:
Naming now doesn't bind it to the navbar which makes it hard to understand when overriding/styling things